include/wx/strvararg.h

```diff
@@ -207,9 +207,17 @@ class WXDLLIMPEXP_BASE wxFormatString
         Arg_Unknown     = 0x8000     // unrecognized specifier (likely error)
     };
 
+#if wxDEBUG_LEVEL
     // Validate all format string parameters at once: the vector contains the
     // format specifiers corresponding to the actually given arguments.
     void Validate(const std::vector<int>& argTypes) const;
+#else
+    // Also provide a trivial implementation of Validate() doing nothing in
+    // this case.
+    inline void Validate(const std::vector<int>& WXUNUSED(argTypes)) const
```

Comment:
This `inline` seems superfluous now, we should be able to remove it, shouldn't we? I.e. write just
```suggestion
    void Validate(const std::vector<int>& WXUNUSED(argTypes)) const
```

Reply:
Done!